submodules/eks/variables.tf

```diff
@@ -55,7 +56,7 @@ variable "kubeconfig_path" {
 
 variable "private_subnets" {
   description = "List of Private subnets IDs and AZ"
-  type        = list(object({ subnet_id = string, az = string }))
+  type        = list(object({ subnet_id = string, az = string, az_id = string }))
```

Comment:
why do we need the az_id if we have the subnet and we already retrieve the az_id from that?